specification/containerservices/resource-manager/Microsoft.ContainerService/preview/2018-08-01-preview/managedClusters.json

```diff
@@ -179,58 +179,6 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/accessProfiles/{roleName}/listCredential": {
```

Comment:
This appears to be deprecation of an API from a stable version. Do you have a deprecation policy in place? Has this been communicated?